experimental/rsocket-src/RSocketServer.cpp

```diff
@@ -92,8 +95,9 @@ void RSocketServer::start(OnAccept onAccept) {
                   folly::Executor& executor) {
         LOG(INFO) << "Going to accept duplex connection";
 
-        // FIXME(alexanderm): This isn't thread safe
-        acceptor_.accept(std::move(conn), connectionHandler_);
+        if (isShutdown_) {
```

Comment:
Oh I remember now, there are no tests that actually exercise `RSocketServer` serving requests.  This is because up until this diff, there was no safe way to run ~RSocketServer after it had started serving connections.